package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "yajsapi",
     "version": "",
-    "description": "NodeJS client for ya-payment",
+    "description": "NodeJS API for Next Golem",
```

Comment:
Golem JS API? Or Golem NodeJS API? `yapapi` is described as `Golem Python API` on pypi.org.